### ISSUE TITLE
GCST changed for the submission id 632c967737db2600018dd446 GCP000432 (PMID:35870639)

### ISSUE THREAD

**Issue by Santhi1901** [EBISPOT/goci#902]
The GCST was changed for the submission of 6790 studies.
It is a user submission, and I added associations and uploaded it in the dep-curation app. It failed to upload as some of the variant id columns were empty. I added those missing rsids and again uploaded the file. This time it got uploaded, but the GCSTs got changed.  I didn't make changes to any of the study tag.

**This has to be done before the next DR.**

I've attached the list of studies with original and changed GCST ids
(https://app.zenhub.com/files/2995118/5cfd7cf2-7c76-4494-ba2b-d959d825b1f7/download)

**Comment by ljwh2** [comment]
GCSTs needs to be fixed asap, root cause can take longer

**Comment by sajo-ebi** [comment]
@Santhi1901 this is very much complex than previously thought ,not a straight forward GCST change,  the issue is the studies have been duplicated probably both unpublished studies & published studies existing in DB , I need to understand the workflow for this submission, how did we get to this scenario of studies getting duplicated in Deposition , this issue had happened once before when we had edited Depo curation without unpublishing the Pmid in Curation App, we can discuss further tomorrow

**Comment by Santhi1901** [comment]
@sajo-ebi 
The user submitted pre-published submission (gwas-subs #613211) and later added the PMID (I am not sure when exactly the PMID was added). This submission appears in the list of pre-published submissions in the Catalog.

The sumstat validation failed multiple times due to md5sum mismatch (https://github.com/EBISPOT/goci/issues/867). 

I uploaded the template with associations through depo-curation app. It failed uploading as some of the rows the variant IDs were missing. I re-uploaded again with all the errors corrected.

This time it got uploaded, but the GCSTs have changed.

**This submission was never published through curation app**
**[Submission history.xlsx](https://github.com/EBISPOT/goci/files/10139656/Submission.history.xlsx)**

**Comment by sajo-ebi** [comment]
The GCST has been updated for the submission & the old unused GCST have been deleted from both Mongo & Curation DB so that reuploading template doesn't retain the old GCST , need to raise a new ticket to add validation in code for submission reset to not allow reset when a submission has been edited in Depo-Curation

**Comment by ljwh2** [comment]
@sajo-ebi to create new ticket then close

**Comment by sajo-ebi** [comment]
Created [213](https://app.zenhub.com/workspaces/gwas-59df823c4a6feb3786810391/issues/gh/ebispot/gwasdepo-deposition-service/213)to track for development purpose
